spec/root_spec.cr

```diff
@@ -0,0 +1,16 @@
+require "./helper"
+
+module PlaceOS::Frontends
+  with_server do
```

Comment:
Top-level describe here, this makes it easier to identify failing specs and conforms better to the rest of the tests.